apps/prairielearn/src/tests/showClosedAssessmentScore.test.ts

```diff
@@ -45,7 +45,7 @@ describe('Exam assessment with showClosedAssessment AND showClosedAssessmentScor
   });
 
   step('enroll the test student user in the course', async () => {
-    await sqldb.queryOneRowAsync(sql.enroll_student_in_course, []);
+    await sqldb.queryAsync(sql.enroll_student_in_course, []);
```

Comment:
I guess I'm not the only one that was relying on the number of course instances in test course in my tests.

Reply:
For clarification, this is not an important part of the test, it is just enrolling a student in a test course instance as set up for later tests. Now that there are two test course instances, the query as written enrolls the test student in both. I don't think that matters, so I made the change to allow more than one row being returned as the fix.